dev-resources/profiles/calibration_concept.jsonld

```diff
@@ -159,10 +159,10 @@
       "type" : "StatementTemplate",
       "inScheme" : "https://xapinet.org/xapi/yet/calibration-concept/v1",
       "prefLabel" : {
-        "en" : "Object Activity Type Template 2"
+        "en-US" : "Object Activity Type Template 2"
```

Comment:
@cliffcaseyyet Curious why the change from `en` to `en-US`?

Reply:
MURICA 🇺🇸 🎆  thats why